manager/resolver/maven/src/main/java/org/jboss/forge/furnace/manager/maven/ClasspathWorkspaceReader.java

```diff
@@ -418,10 +418,13 @@ private Artifact createFoundArtifact(final File pomFile)
             if (parent != null)
             {
                Xpp3Dom relativePathNode = parent.getChild("relativePath");
-               String relativePath = (relativePathNode == null) ? "../pom.xml" : relativePathNode.getValue();
-               File parentPom = pomFile.getParentFile().toPath().resolve(relativePath).toFile();
-               if (parentPom.isFile())
-                  result = createFoundModules(parentPom);
+               if ( relativePathNode != null && !(relativePathNode.getValue() == null)  ) 

```

Comment:
This looks better:
    if ( relativePathNode != null && relativePathNode.getValue() != null )